augur/routes/pull_request_reports.py

```diff
@@ -567,11 +590,14 @@ def average_commits_per_PR():
     @server.app.route('/{}/pull_request_reports/average_comments_per_PR/'.format(server.api_version), methods=["GET"])
     def average_comments_per_PR():
 
-        now = datetime.datetime.now()
```

Comment:
What happens if somebody provides a start date, but not an end date? Is it defaulted to "now" elsewhere?

Reply:
If someone provides a start date but no end date, the end date defaults to now. This made me think that I should add an error check to make sure the start_date is before the end_date though.